R/sqlQueries.R

```diff
@@ -74,14 +77,15 @@
                        ')'):: DATERANGE
```

Comment:
These typecasts with :: DATERANGE with space seem weird to me. This might work, but if both works i'd rather go with the 'somerange_as_string'::DATERANGE notation.